api/src/services/export/user-devices-mapper.js

```diff
@@ -0,0 +1,22 @@
+const db = require('../../db');
+
+module.exports = async () => {
+  const { rows } = await db.medicUsersMeta.query('users-meta/device_by_user', { group: true });
+  return rows.map(doc => {
+    const user = doc.key;
+    const date = doc.value.date;
+    const deviceId = doc.value.device.deviceId;
+    const webview = /Chrome\/(\d+\.\d+)/.exec(doc.value.device.userAgent)?.[1];
```

Comment:
I believe it will, good catch! I focused only on CHWs devices that are running the CHT in a webview but it makes sense that an admin would want to know if let's say a supervisor is using an unsupported version of Firefox.

What do you think of renaming this field from `webview` to `browser` and setting the value to `"$browser_name $major_version"`?
It would be safer to rely on a reliable useragent parser like bowser instead of dealing with edge cases ourselves. It's the same dependency we use to detect unsupported browsers on the login page and in the BrowserDetectorService. The `user-devices-mapper.js` code would look like this:
```js
const Bowser = require('bowser');

const { rows } = await db.medicUsersMeta.query('users-meta/device_by_user', { group: true });
return rows.map(doc => {
  // ...
  const { browser } = Bowser.parse(doc.value.device.userAgent);
  return {
    // ...
    browser: `${browser.name} ${browser.version}`,
    // ...
  };
});
```

Reply:
+1000 for `bowser`!  I was a little unsure about combining both `name` and `version` into the same field. :thinking: Trying to decide how a consumer would be most likely to use this data....  My current thought is that what you have done makes the most sense and keeps everything nice and simple.  It is not like consumers can do a simple number/alpha sort on browser versions to "get the earliest" anyway.  So, including the browser name in the string is not going to add much of an extra "burden" to processing the data.

Reply:
I was initially had the CSV formatted export in mind but since we're giving back JSON it'd be fine to nest data. JSON -> CSV tools know how to deal with this scenario anyway.